packages/screenshots/Classes/Configuration/Configuration.php

```diff
@@ -221,7 +221,34 @@ public function createBasicConfig(): void
                 ],
                 'Examples' => [
                     'screenshots' => [
-                        [
+                        "_default"=> [
```

Comment:
Remove that _default section as it does not provide additional insights. This is copied from a production screenshots.json but has no additional benefit here.

Reply:
In my eyes it has the same benefit like in the introduction section. This should be the standard way of doing these kind of settings.